backend/app/store/image/bolt_store.go

```diff
@@ -0,0 +1,154 @@
+package image
+
+import (
+	"bytes"
+	"context"
+	"encoding/binary"
+	"io"
+	"io/ioutil"
+	"path"
+	"time"
+
+	bolt "github.com/coreos/bbolt"
+	log "github.com/go-pkgz/lgr"
+	"github.com/pkg/errors"
+)
+
+type Bolt struct {
```

Comment:
None of public struct or public methods commented. I wonder how linter missed it, will check. However all exposed things should be command like:

// Bolt implements the store for images keeping data in 3 buckets ....